examples/make_import/build.go

```diff
@@ -55,3 +62,48 @@ func Env() (err error) {
 
 	return
 }
+
+type Help mg.Namespace
+
+// Envs outputs the current environment configuration
+func (Help) Envs() (err error) {
+	m := root.Target("help:envs")
+	defer func() { m.Complete(err) }()
+
+	return common.WriteEnvs(root.Env(), os.Stdout)
+}
+
+// importEnvFromMakefile invokes `make` to generate configuration for this mage script.
+// The script outputs a set of environment variables prefixed with `MAGNET_` which
+// are used as default values for the configuration variables defined by the script.
+func importEnvFromMakefile() (env map[string]string) {
```

Comment:
All the parsing should be re-useable library code and not duplicated into each project.

Reply:
Addressed in 84d297b